src/decoder.rs

```diff
@@ -258,6 +258,39 @@ impl Decoder {
         map_sys_error!(result => ())
     }
 
+    #[cfg(feature = "dred")]
+    #[inline]
+    ///Configures computational complexity of the decoder.
+    ///
+    ///Supported values are from 0 to 10, 10 meaning highest complexity.
+    ///Values of 5 or above will use deep packet loss concealment.
+    pub fn set_complexity(&mut self, value: u8) -> Result<(), ErrorCode> {
+        if value > 10 {
```

Comment:
There is no need to duplicate this check, C code will validate it and return error just fine too